express-basic/list/utility.js

```diff
@@ -0,0 +1,33 @@
+function someFunc(colors){
```

Comment:
functions and variables should always have meaningful names - this is not a good name as it does not tell me what the function does